frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -7,6 +7,12 @@ import { useDispatch, connect } from 'react-redux';
 import { sortable } from '@patternfly/react-table';
 import { useTranslation } from 'react-i18next';
 import { ChartDonut } from '@patternfly/react-charts';
+import { useExtensions } from '@console/plugin-sdk';
+import {
+  isPVCAlert,
+  isPVCCreateProp,
+  isPVCStatus,
+} from '@console/dynamic-plugin-sdk/src/extensions/pvc';
```

Comment:
Once #8224 is merged this should be simply import from `@console/dynamic-plugin-sdk`.